cmd/next.go

```diff
@@ -61,6 +62,10 @@ func nextPipeline(afs afero.Afero, writer io.Writer, part string, prerelease, me
 		return err
 	}
 
+	if part != "patch" && part != "minor" && part != "major" {
```

Comment:
I would use a `map[string]struct{}` for readability

Reply:
see below: used cobra lib now.